sriovnet_test.go

```diff
@@ -117,6 +150,43 @@ func TestGetPfPciFromVfPciError(t *testing.T) {
 	assert.Equal(t, "", pf)
 }
 
+func TestGetPfPciFromAux(t *testing.T) {
```

Comment:
can you also add tests for `GetSfIndexByAuxDev` ?

Reply:
done